datalake/src/main/java/com/sequenceiq/datalake/flow/create/SdxCreateActions.java

```diff
@@ -145,7 +150,10 @@ protected SdxContext createFlowContext(FlowParameters flowParameters, StateConte
 
             @Override
             protected void doExecute(SdxContext context, SdxCreateFailedEvent payload, Map<Object, Object> variables) throws Exception {
-                sdxService.updateSdxStatus(payload.getResourceId(), SdxClusterStatus.PROVISIONING_FAILED);
+                Exception exception = payload.getException();
+                LOGGER.info("Update SDX status to delete failed {}", payload.getResourceId(), exception);
```

Comment:
Copy-paste: change "delete" to "provisioning".

Reply:
Thanks, fixed!